code/testapp/src/main/java/com/adobe/marketing/mobile/services/ui/TestAppUIServices.java

```diff
@@ -11,13 +11,6 @@
 
 package com.adobe.marketing.mobile.services.ui;
 
-import com.adobe.marketing.mobile.services.ui.AlertSetting;
```

Comment:
Are you planning to fix this app in android PR ?

Reply:
Yes, I will fix the Java test app in a separate PR to avoid clutter.